source/ranges.tex

```diff
@@ -901,24 +901,24 @@
 
 \rSec2[range.view]{Views}
 
-\indexlibrary{\idxcode{View}}%
+\indexlibrary{\idxcode{view}}%
 \begin{itemdecl}
 template<class T>
-  concept View =
+  concept view =
     range<T> && semiregular<T> && enable_view<T>;
 \end{itemdecl}
 
 \begin{itemdescr}
-% FIXME: This should explicitly say when View is modeled.
+% FIXME: This should explicitly say when view is modeled.
 \pnum
-The \libconcept{View} concept specifies the requirements of a \libconcept{range} type
+The \libconcept{view} concept specifies the requirements of a \libconcept{range} type
 that has constant time copy, move, and assignment operators; that is, the cost of
 these operations is not proportional to the number of elements in the
-\tcode{View}.
+\tcode{view}.
```

Comment:
IMO this should be `the number of elements in the \libconcept{view}`. A concept doesn't have elements, but an instance of some object type that models `range` (or a refinement thereof like `view`) does. Adding the word "concept" here wouldn't be consistent with the usage of the colloquialism and would cause confusion.